examples/tailwindcss/lynx.config.ts

```diff
@@ -0,0 +1,8 @@
+import { pluginQRCode } from "@lynx-js/qrcode-rsbuild-plugin";
+import { pluginReactLynx } from "@lynx-js/react-rsbuild-plugin";
+import { defineConfig } from "@lynx-js/rspeedy";
+import { pluginTailwindCSS } from "rsbuild-plugin-tailwindcss";
+
+export default defineConfig({
+  plugins: [pluginReactLynx(), pluginQRCode(), pluginTailwindCSS()],
+});
```

Comment:
I think this could run on Web Platform, maybe we should add `environments.web` and `environments.lynx` (which would enable Web Preview on the Website)